server/datastore/mysql/schema.sql

```diff
@@ -958,7 +958,7 @@ CREATE TABLE `labels` (
   `updated_at` timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
   `name` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
   `description` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
-  `query` mediumtext COLLATE utf8mb4_unicode_ci NOT NULL,
+  `query` text COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
Why did this change? Seeing this change a few other places.

Reply:
@iansltx I verified that this change has to do with the change away from `utf8` (I changed the character set to be `utf8` in the `labels` table creation migration and it switched the column type back to `MEDIUMTEXT`.

I can't find any documentation around this behavior, but I think this is OK (especially since we'll be aligning with the type that's set in the migrations, which is `TEXT`).